Touk_BookingApp/src/main/java/com/lc/touk_bookingapp/enums/TicketType.java

```diff
@@ -0,0 +1,17 @@
+package com.lc.touk_bookingapp.enums;
+
+public enum TicketType {
+    ADULT(25),
+    STUDENT(18),
+    CHILD(12.5);
+
+    private final double price;
```

Comment:
a jak biznes postanowi zmienić stawki to co będzie trzeba zrobić w naszej aplikacji? czy będzie to wygodne?